zwave_js/CHANGELOG.md

```diff
@@ -1,5 +1,41 @@
 # Changelog
 
+## 0.1.91
+
+### Breaking changes

```

Comment:
I take it that the "heal" -> "rebuild routes" change haven't found their way into HA yet?

Reply:
it'll be handled transparently in 2023.10. It's not worth mentioning here because it doesn't impact device functionality